internal/debug/flags.go

```diff
@@ -87,16 +88,16 @@ var Flags = []cli.Flag{
 	memprofilerateFlag, blockprofilerateFlag, cpuprofileFlag, traceFlag,
 }
 
-// glogger is the glog handler used by Geth, allowing the debug APIs to modify
-// verbosity levels, vmodules and backtrace locations.
-var glogger = log.NewGlogHandler(log.StreamHandler(os.Stderr, log.TerminalFormat()))
+var (
+	usecolor = term.IsTty(os.Stderr.Fd()) && os.Getenv("TERM") != "dumb" && runtime.GOOS != "windows"
```

Comment:
If you use 

`glogger  = log.NewGlogHandler(log.StreamHandler(colorable.NewColorableStderr(), log.TerminalFormat(usecolor)))`

it should work ok on Windows.